docs/concepts/network-lifecycle.md

```diff
@@ -24,3 +24,20 @@ The valid `NetworkID` range can be configured using the `apiserver`s
 
 When deleting a `Network`, the corresponding `NetworkID` is cleaned up
 alongside the claiming `Network`.
+
+## Network Peering
+
+When creating network peering both `ironcore` `Network`s has to specify matching, 
+`spec.peerings` referencing each other respectively. A `ironcore` `Network` can be peered with 
+multiple `network`s in any namespcae.
+
+Once specified `apinetlet` `NetworkPeeringController` validates if all the specified `network`s are in 
```

Comment:
Do we really need an own reconciler for the `Network` peering or wouldn't it be just enough to extend the existing `Network` reconciler to ensure the peering information in the `Network` in the `apinet` group and update the `ironcore` Status once the peering was established.

Reply:
Network peering is happening only with `Available` state networks, so I think we should have different reconciler for network peering. If we do it in existing `Network` controller we have to wait till both networks come `Available` and then do peering in same reconcile loop which will become so lengthy.
@balpert89 do you have different thoughts for new controller?